lib/api/sfApi.js

```diff
@@ -117,6 +118,7 @@ function setAxiosDefaults(firmId) {
   if (firmTokens) {
     axios.defaults.baseURL = `${baseURL}/api/v4/f/${firmId}`;
     axios.defaults.headers.common[
+      "User-Agent" = `Silverfin CLI/${pkg.version}`,
       "Authorization"
     ] = `Bearer ${firmTokens.accessToken}`;
```

Comment:
The previous code doesn't work, it will break the CLI. The User-Agent should be outside `common`

```suggestion
    axios.defaults.baseURL = `${baseURL}/api/v4/f/${firmId}`;
    axios.defaults.headers["User-Agent"] = `Silverfin CLI/${pkg.version}`;
    axios.defaults.headers.common[
      "Authorization"
    ] = `Bearer ${firmTokens.accessToken}`;
```

Reply:
Moved the line outside common 
Also used `silverfin-cli` naming instead so it also matches our project name